src/Table/tableUtil.tsx

```diff
@@ -0,0 +1,19 @@
+import { StyledArrowDown, StyledArrowUp, StyledUnsorted } from './tableStyles';
+
+export interface Column {
+    disableSortBy: boolean;
+    isSorted: boolean;
+    isSortedDesc: boolean;
+}
+
+export const getSortBySVG = ({disableSortBy, isSorted, isSortedDesc}: Column) => {
+    if(disableSortBy){
+        return null
+    }
+    return (        
+        (!disableSortBy && (isSorted ? 
+            (isSortedDesc ? <StyledArrowDown aria-label='arrow-down' className='sort-icon' /> : <StyledArrowUp aria-label='arrow-up' className='sort-icon' />) 
```

Comment:
No change required, by just my recommendation. Nested ternary statements are often difficult to easily read what's going on. Since you went through the trouble of moving `getSortBySVG` into its own logic, you can go a step further and extract line 15 into another function (like `getSortDirectionSVG`). Then this would read easier imo:

```tsx
return isSorted ? getSortDirectionSVG(isSortedDesc) : <StyledUnsorted aria-label='unsorted-icon' className='sort-icon' />;
```